composeApp/src/commonMain/kotlin/org/ooni/probe/ui/choosewebsites/ChooseWebsitesViewModel.kt

```diff
@@ -104,7 +104,6 @@ class ChooseWebsitesViewModel(
                     RunSpecification.Full(
                         tests = listOf(
                             RunSpecification.Test(
-                                source = RunSpecification.Test.Source.Default("websites"),
```

Comment:
We should fetch it from the DB, or hardcode the websites descriptor ID somewhere.